### DISCUSSION TITLE
ENIGMA 2

### DISCUSSION THREAD

**Discussion by Yamaha0014** [PRProd/HA-Firemote#294] [New Device Family Support Request]
Hi Is it possible add remote controler to enigma 2 reciver ? Example Dreambox.

**Comment by PRProd** [comment]
Hello @Yamaha0014!

I've been making the latest Firemote additions based off of community requests just like yours.  There is a poll in discussion #41 where votes are tallied and I announce the winners based off of popularity and off of my ability to get my hands on the device.  (I don't want to release something before I can test it for myself)

[Roku is the winner of the latest round of voting](https://github.com/PRProd/HA-Firemote/discussions/41#discussioncomment-7801488) and I am currently in a development cycle of [beta release testing](https://github.com/PRProd/HA-Firemote/discussions/275) which I hope to complete soon.   Once that is done, I will reset the poll so voting can start all over again.

One quick thing to note: as stated in the [Firemote wiki](https://github.com/PRProd/HA-Firemote/wiki#how-does-firemote-send-commands-to-my-devices)

![image](https://github.com/PRProd/HA-Firemote/assets/25486599/05b102a4-eaa5-46c9-a129-8f88e351122b)

With this in mind, I will need you to confirm yes or no that the device family you are requesting is able to be completely controlled already through Home Assistant.  An official integration is REQUIRED FIRST before Firemote can talk to your device.

So, what should you do in the meantime?  Three things:
 1. Add your device request [into the poll's discussion thread](https://github.com/PRProd/HA-Firemote/discussions/41#discussioncomment-5293057)
 2. Be sure to cast your vote when the poll is reset back to zero.  For now, you could vote for "other" but be aware that all votes will be reset soon. so you will have to vote again
 3. Add as many important details as you can about this device into [this discussion thread](https://github.com/PRProd/HA-Firemote/discussions/294).  Include things like:
    - Product links
    - Product Pictures
    - Remote Pictures
    - Anything else you can find
